distuv/distribution_test.go

```diff
@@ -9,135 +9,214 @@ import (
 	"sort"
 	"testing"
 
+	"github.com/btracey/quad"
```

Comment:
Good catch. Done.

I can't win with goimports and quad. I deleted btracey/quad from existence, then it wanted github.com/gonum/quadold (which I had kept because of other branches). I changed the package name in quadold, and then it wanted gonum.org/pkg/quad